helm_drydock/drivers/node/maasdriver/driver.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
This naming is more expected, but is not aligned with `class NodeDriver` as mentioned above.